setup.py

```diff
@@ -3,10 +3,11 @@
 with open('requirements.txt') as f:
     required = f.read().splitlines()
 
-setup(name='RFEst',
-      version='2.0.2',
+setup(
+      name='RFEst',
+      version='2.1.0',
```

Comment:
I just checked what's the best way to track Python package versions, turns out that it's a mess (https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package). 

With what we did here, we cannot access the version in .ipynb via `import rfest; rfest.__version__`, because we never defined one. The easy way is to do it is to add `__version__ = '2.1.0'` in `rfest/__init__.py`, but then we need to maintain the same info two files (`setup.py` and `__init__.py`). Or we add a `version.py` file, define `__version__` there, and read it to both `setup.py` and `__init__.py`.